app/models/katello/errata_status.rb

```diff
@@ -9,17 +10,30 @@ def self.status_name
       N_("Errata")
     end
 
+    def errata_status_installable
+      Setting[:errata_status_installable]
+    end
+
+    def profiles_reporter_package_name
```

Comment:
This check isn't 100% accurate, but is more precise than it was. It could be improved by checking that the subscription-manager that's installed is >= the version which introduced package+repo profile reporting